docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -146,7 +146,7 @@ SHOW TABLE EXTENDED  LIKE `employe*`;
 +--------+---------+----------+---------------------------------------------------------------+
   
 -- show partition file system details
-SHOW TABLE EXTENDED  IN default LIKE `employee` PARTITION (`grade=1`);
+SHOW TABLE EXTENDED  IN default LIKE `employee` PARTITION (grade=1);
```

Comment:
Could you remove this backquote on employee, @Udbhav30 ?

Reply:
I think backquote on employee is required. I get parse error without backqoute

Reply:
@dongjoon-hyun sorry my bad you are correct it is `'employee'` instead of backqoute.
This whole time i was also using  `'employee'` . I will update this too